tests/plugins/test_orchestrate_build.py

```diff
@@ -480,7 +481,16 @@ def test_orchestrate_build_choose_clusters(tmpdir, clusters_x86_64,
         assert plat_annotations['build']['cluster-url'] == 'https://chosen_{}.com/'.format(platform)
 
 
-def test_orchestrate_build_exclude_platforms(tmpdir):
+@pytest.mark.parametrize(('platforms', 'platform_exclude', 'platform_only'), [
```

Comment:
This test should probably specify a list of platforms that are expected to be set, instead of repeating logic inside the unit test.